<?xml version="1.0" encoding="UTF-8"?>
<!-- progress.ui
  Bluemoon - a Bluetooth manager designed for LXQt desktop environment
  https://github.com/sklins/BlueMoon
     
  Copyright: 2016, Bluemoon Team
  Authors:
  Sofiia Sandomirskaya <ssandomirskaya@gmail.com>
  Elina Gilmanova <linkegilmanova@gmail.com>
  Denis Pavlov <dspavlov_1@edu.hse.ru>
  Linh Chi Tran <lc.tinatran@gmail.com>
  
  This program is free software; you can redistribute it and/or modify 
  it under the terms of the GNU Lesser General Public License as published 
  by the Free Software Foundation; either version 2.1 of the License, 
  or (at your option) any later version.
  
  This program is distributed in the hope that it will be useful,
  but WITHOUT ANY WARRANTY; without even the implied warranty of
  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
  Lesser General Public License for more details.
  
  You should have received a copy of the GNU Lesser General
  Public License along with this library; if not, write to the
  Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
  Boston, MA 02110-1301 USA 
-->
<ui version="4.0">
 <class>Progress</class>
 <widget class="QDialog" name="Progress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>216</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>408</width>
    <height>216</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>408</width>
    <height>216</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>File Transfer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/scalable_bluemoon.svg</normaloff>../icons/scalable_bluemoon.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="chooseFileButton">
       <property name="text">
        <string>Choose</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLineEdit" name="fileNameLineEdit">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="transferButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Start transfer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Filename</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="minimumSize">
      <size>
       <width>390</width>
       <height>0</height>
      </size>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
